maven-core/src/main/java/org/apache/maven/DefaultMaven.java

```diff
@@ -19,6 +19,25 @@
  * under the License.
  */
 
+import javax.inject.Inject;
```

Comment:
Could you please revert this change as it seems unrelated to what you want to achieve.